README.md

```diff
@@ -30,28 +30,19 @@ In this repo the fuzzers have the following structure:
 
 ## User Guide
 
-### 1. create the docker container 
-* Use the command  `docker build -t ready-to-fuzz-image .` inside this project to create a docker image named **ready-to-fuzz-image** from the dockerfile in the project.
+### 1. Create the docker container 
+To create and run the docker container ready to run the fuzzers use the command:
+`make docker-up`
 
-* run the command `docker run -it --rm --cap-add=SYS_PTRACE --security-opt seccomp=unconfined ready-to-fuzz-image bash` to run the container from the image previously created.
 
 ### 2. Run a fuzzer 
 
-Depending on the fuzzer use the following commands:
+Each fuzzer has its own Readme with specific instructions to run each of them.
```

Comment:
```suggestion
Each fuzzer has its own README with specific instructions to run each of them.
```

Reply:
Also I think it would be better to add links to every fuzzer folder and its README

Reply:
done 👍